src/hooks/useTikTokAuth.ts

```diff
@@ -0,0 +1,155 @@
+import React from 'react'
+
+import moment from 'moment'
+
+// Polyfills
+import 'whatwg-fetch'
+import 'url-search-params-polyfill'
```

Comment:
wow we need a polyfill for this one!?

Reply:
Not entirely sure but I copied it from the TwitterAuth component to be safe